Source/Core/Core/DSP/DSPDisassembler.cpp

```diff
@@ -132,7 +133,8 @@ std::string DSPDisassembler::DisassembleParameters(const DSPOPCTemplate& opc, u1
       break;
 
     default:
-      ERROR_LOG_FMT(DSPLLE, "Unknown parameter type: {:x}", static_cast<u32>(opc.params[j].type));
+      ERROR_LOG_FMT(DSPLLE, "Unknown parameter type: {:x}",
+                    Common::ToUnderlying(opc.params[j].type));
```

Comment:
This technically changes a cast to `u32` to a cast to `int`, because `partype_t` is not typed `u32`. I don't think this matters much, but just noting.